Library/Homebrew/rubocops/checksum_cop.rb

```diff
@@ -0,0 +1,61 @@
+require_relative "./extend/formula_cop"
+
+module RuboCop
+  module Cop
+    module FormulaAudit
+      class Checksum < FormulaCop
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          %w[stable devel head].each do |name|
+            next unless spec_node = find_block(body_node, name.to_sym)
+            _, _, spec_body = *spec_node
+            audit_checksums(spec_body, name)
+            if name == "stable"
+              resource_blocks = find_blocks(body_node, :resource) +
+                                find_all_blocks(spec_body, :resource)
```

Comment:
@I think you also want to always add `body_node` as it's effectively a `resource` block too. Similarly with root level `patch` blocks.